ts/store/reducers/content.ts

```diff
@@ -166,13 +162,10 @@ export const idpContextualHelpDataFromIdSelector = (id: IdentityProviderId) =>
  * return an option with screen contextual help data if they are loaded and defined
  * @param id
  */
-export const screenContextualHelpData = (navState: NavigationState) =>
-  createSelector<
-    GlobalState,
-    pot.Pot<ContextualHelp, Error>,
-    Option<ScreenCHData>
-  >(contextualHelpDataSelector, contextualHelpData => {
-    return pot.getOrElse(
+export const screenContextualHelpDataSelector = createSelector(
```

Comment:
```suggestion
export const screenContextualHelpDataSelector = createSelector<
  GlobalState,
  pot.Pot<ContextualHelp, Error>,
  NavigationState,
  Option<ScreenCHData>
>
```
using type helps make the result of the select strong typed and could help to avoid mistakes

Reply:
ok. Updated within https://github.com/pagopa/io-app/pull/1898/commits/732737daf8e67773abbeb723ee18a72f1004a450